zooniverse/front-end-monorepo
# Fix invert button for multi-frame subjects and line color consistency

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- closes #3328 

## Describe your changes
- add invert status from subject viewer store to MultiFrameViewerContainer
- add SVGImage component with svg filter to invert colors, image applies/defines filter on svg image per invert property
- ^ similar to [PFE SVGImage](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/components/svg-image.jsx)

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - (multi-frame subject, transcription lines) - [⚠️ **CAUTION** ⚠️ live production project] - https://local.zooniverse.org:8080/?project=humphrydavy%2Fdavy-notebooks-project&env=production&demo=true&workflow=18575
  - (single image frame, and text, subject) - https://local.zooniverse.org:8080/?project=1952&workflow=3602&demo=true
- What user actions should my reviewer step through to review this PR?
  - if drawing/transcription task, draw a few lines, noting colro
  - click Invert button in ImageToolbar
  - image color should invert, line colors should remain the same
  - click Invert button in ImageToolbar
  - image color should revert to original/normal, line colors should remain the same
- Which storybook stories should be reviewed?
  - http://localhost:6006/?path=/story/image-toolbar-image-toolbar--with-multi-frame-viewer&args=dark:true&globals=locale:en;locales.en:English;locales.test:Test+Language
- Are there plans for follow up PR’s to further fix this bug or develop this feature? the MultiFrameViewer will be further refactored as per #3210 

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook


## Bug Fix
